en/ui-system/components/engine/index.md

```diff
@@ -1,8 +1,8 @@
 # UI Architecture
 
-The UI uses a tree-based rendering structure, and the entire UI is rendered based on the Canvas node (a node with `Canvas`) as the root node. That is, the final root node of the UI node must be the Canvas node before it can be rendered by the Canvas. The `UITransform` is the necessary condition that each UI node must have for a click or alignment strategy, etc. to take effect.
+The UI rendering is based on tree structure. The **Canvas** node (a node with `Canvas component`) is the root of UI. **All ui nodes must be under the Canvas node to be rendered**. The `UITransform` is the necessary condition that each UI node must have for a click or alignment strategy, etc. to take effect.
```

Comment:
The UI rendering is based on tree structure.
-->
The UI rendering is based on a tree structure.